deltachat-ios/Controller/MailboxViewController.swift

```diff
@@ -1,68 +0,0 @@
-import UIKit
```

Comment:
cool that we never renamed that to `ContactRequestsViewController` - would have been a waste of time :)

”Wisdom of a Stone" :)

Reply:
🤣